src/ui/public/styles/bootstrap/navs.less

```diff
@@ -87,8 +87,8 @@
       border: 1px solid transparent;
       border-radius: @border-radius-base @border-radius-base 0 0;
       &:hover {
-        border-color: @globalColorLightGray;
```

Comment:
I understand the logic of keeping the bootstrap vars bootstrap vars, but should we at least shift the top level bootstrap vars to our globals? Otherwise these currently get a bunch of pure grays.